fees/binance-staked-eth.ts

```diff
@@ -12,6 +13,21 @@ const methodology = {
   Revenue: '10% staking rewards are charged by Binance.'
 }
 
+const breakdownMethodology = {
```

Comment:
maybe if breakdown is missing, I can automatically use the methodology field to show this info on imcome statements, this way, we avoid duplication of code?